{"uuid": "d68a5380-3fe7-48d2-9cd2-5e923777a2f0", "children": ["ddddca46-bb03-4fda-87af-a43bf6cf3224", "5190bfab-5d8d-4e6e-8af1-c1ed75ea8c35", "291ef1f9-0d18-4118-8117-d078c69fd642", "687c8a49-3e0c-49b8-9cbe-f3608946a4b1", "aa83ef97-1ec8-45c6-bb10-5d10645834a8"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1586835753384, "stop": 1586835753384}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'mail_pass' is not defined\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 707, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/sonic/Project/Python/Pytest/scripts/test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"/Users/sonic/Project/Python/Pytest/uti/EmailHandler.py\", line 59, in send_email\n    smtp_obj.login(mail_user, mail_pass)\n"}, "start": 1586835753538, "stop": 1586835765322}], "start": 1586835753384, "stop": 1586835765323}